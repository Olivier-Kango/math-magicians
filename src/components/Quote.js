import React from 'react';

const h3 = {
  margin: '25vh 0',
  textAlign: 'center',
  color: '#00004B',
};

const Quote = () => (
  <div>
    <h3 style={h3} className="h-quote">
      Mathematics is not about numbers, equations, computations, or algorithms:
      <br />
      It is about understanding.
      <br />
      <br />
      <i>William Paul Thurston</i>
    </h3>
  </div>
);

export default Quote;
