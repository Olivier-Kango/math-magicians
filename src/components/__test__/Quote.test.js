import { render, screen } from '@testing-library/react';
import Quote from '../Quote';

describe('Testing the Home page:', () => {
  test('should render', () => {
    render(<Quote />);
    const textElement = screen.getByText(/Mathematics is not about numbers, equations, computations, or algorithms:/i);
    expect(textElement).toMatchSnapshot();
  });
});
