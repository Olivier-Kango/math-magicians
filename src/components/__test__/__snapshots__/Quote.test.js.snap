// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Quote page: should render 1`] = `
<h3
  style="margin: 25vh 0px; text-align: center; color: rgb(0, 0, 75);"
>
  Mathematics is not about numbers, equations, computations, or algorithms:
  <br />
  It is about understanding.
  <br />
  <br />
  <i>
    William Paul Thurston
  </i>
</h3>
`;
